[#3942] Instructor's Student Records Page: Feedback session panel chevron barely visible #12668

Tim-Siu opened the pull request:
<!-- Before opening a PR, please ensure you have read our contributor guidelines -->
<!-- at https://teammates.github.io/teammates/process.html#step-4-submit-a-pr. -->

<!-- PR title: Copy-and-paste the name of the issue this PR is fixing, -->
<!-- and include the issue number in front in square brackets. -->
<!-- e.g. [#3942] Remove unnecessary System.out.printlns from Java files -->

<!-- Add the issue number to the "Fixes" keyword below. -->
Fixes #12668

**Outline of Solution**

In this PR, I've updated the color of the `tm-panel-chevron` from white to black. This change aims to improve the visibility of the chevron on white backgrounds, based on the feedback and suggestions from the community.

Changes made:
- Modified the `chevronColor` in `PanelChevronComponent` to black.
- Checked and ensured that this color change integrates well with the existing user interface.

I've tested this change and believe it enhances the visibility of the chevron without impacting other design aspects. I'm open to any further suggestions or feedback.

<img width="1379" alt="Screenshot 2023-12-20 at 17 02 17" src="https://github.com/TEAMMATES/teammates/assets/61866948/095867a3-a479-4f5b-8599-15c529f9a656">

weiquu commented on src/web/app/pages-instructor/instructor-student-records-page/instructor-student-records-page.component.html:
```diff
@@ -6,7 +6,7 @@ <h2>Records in feedback sessions</h2>
         <strong class="text-break">Feedback Session: {{ session.feedbackSession.feedbackSessionName }}</strong>
         <div class="card-header-btn-toolbar">
-          <tm-panel-chevron [isExpanded]="!session.isCollapsed"></tm-panel-chevron>
+          <tm-panel-chevron [isExpanded]="!session.isCollapsed" [chevronColor]="'black'"></tm-panel-chevron>
```
I think let's just go with `chevronColor="black"` without the [] property binding, since we are just assigning a static value (this is also aligned with the `instructor-session-result-question-view` component)

weiquu reviewed (changes requested):
Looks good! 1 small nit

Tim-Siu commented:
Updated the PR as per the review suggestions. I've changed the chevron color assignment to a static value (`chevronColor="black"`) without property binding, aligning it with the implementation in the instructor-session-result-question-view component. Looking forward to any further feedback or suggestions.

weiquu reviewed (approved):
LGTM

jasonqiu212 reviewed (approved):
LGTM! Thanks for contributing to TEAMMATES!
